src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -538,66 +545,85 @@ def generate_porosity_permutations(
         material_name : str
             Material name.
         laser_powers : list[float]
-            Laser powers (W) to use for porosity simulations.
+            Laser powers (W) to use for porosity simulations. Valid values
+            are from :obj:`MIN_LASER_POWER <MachineConstants.MIN_LASER_POWER>`
+            to :obj:`MAX_LASER_POWER <MachineConstants.MAX_LASER_POWER>`.
         scan_speeds : list[float]
-            Scan speeds (m/s) to use for porosity simulations.
-        size_x : float, default: :obj:`DEFAULT_SAMPLE_SIZE`
+            Scan speeds (m/s) to use for porosity simulations. Valid values are from
+            :obj:`MIN_SCAN_SPEED <MachineConstants.MIN_SCAN_SPEED>` to
+            :obj:`MAX_SCAN_SPEED <MachineConstants.MAX_SCAN_SPEED>`.
+        size_x : float, default: :obj:`DEFAULT_SAMPLE_SIZE <PorosityInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the porosity sample in the x direction.
-            Valid values are between 0.001 and 0.01.
-        size_y : float, :obj:`DEFAULT_SAMPLE_SIZE`
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <PorosityInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <PorosityInput.MAX_SAMPLE_SIZE>`.
+        size_y : float, :obj:`DEFAULT_SAMPLE_SIZE <PorosityInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the porosity sample in the y direction.
-            Valid values are between 0.001 and 0.01.
-        size_z : float, :obj:`DEFAULT_SAMPLE_SIZE`
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <PorosityInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <PorosityInput.MAX_SAMPLE_SIZE>`.
+        size_z : float, :obj:`DEFAULT_SAMPLE_SIZE <PorosityInput.DEFAULT_SAMPLE_SIZE>`
             Size (m) of the porosity sample in the z direction.
-            Valid values are between 0.001 and 0.01.
-        layer_thicknesses : list[float], None
+            Valid values are from :obj:`MIN_SAMPLE_SIZE <PorosityInput.MIN_SAMPLE_SIZE>`
+            to :obj:`MAX_SAMPLE_SIZE <PorosityInput.MAX_SAMPLE_SIZE>`.
+        layer_thicknesses : list[float], default None
```

Comment:
It looks like "default: None" and "default None" are considered the same according to this documentation: https://numpydoc.readthedocs.io/en/latest/format.html#sections. I'm just mentioning this because src/ansys/additive/core/parametric_study/parametric_utils.py uses "default: None" if you want it to be the same across files

Reply:
@klmcadams and @pkrull-ansys I'd confirm that they are rendered the same in the documentation just in case Sphinx (or our Ansys Sphinx template) render them differently.

Reply:
changed for consistency